src/ai/ai.cpp

```diff
@@ -3,6 +3,41 @@
 
 namespace ai {
 
+static inline float estimate_strength(sys::state& state, dcon::nation_id n) {
```

Comment:
Again, why are these functions static? Or inline for that matter, since they are in a cpp file?